en/docs/install-and-setup/configuring-api-manager-for-ob.md

```diff
@@ -79,36 +79,75 @@ database server, and the JDBC driver.
    for access without PSU involvement per day. This enables throttling requests according to the `frequencyPerDay` value provided 
    in accounts initiation request. For a request to be throttled, the TPP should initiate the request.
 
-    ``` toml
-    [open_banking_berlin.consent.freq_per_day]
-    enable = true
-    ```
-    
+    1. To configure the custom throttling policy, enable the `frequencyPerDay` property as follows:
```

Comment:
Let's say, "enable the following config" instead of "enable the "frequencyPerDay" property". @isharailanga , WDYT?

Reply:
either way is correct. we can refer to the other occurrences in this page and follow the same style of consistency. @DinithiDiaz

Reply:
Fixed in bfc2d00e04fd6b0d079f3d653be008221b2520dd.